app/src/main/java/com/example/androidapp/ui/tasks/TasksFragment.java

```diff
@@ -63,7 +63,7 @@ public void onDestroyView() {
     public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
        Log.i("TaskFragment", "You clicked Item: " + id + " at position:" + position);
        Task selected_task= list_of_tasks.get(position);
-        if(selected_task.isCompleted()==false)
+        if(selected_task.getCompleted()==false)
```

Comment:
Leave this as "isCompleted" and change the name of the method in Task.java instead